### DISCUSSION TITLE
Regarding filament performance

### DISCUSSION THREAD

**Discussion by devefy-ashish** [filamentphp/filament#14217] [Help]
### Package

Panel builder

### Package Version

3

### How can we help you?

I am working with laravel filament and building my first project using filament. I am facing low speed of filament dashboard performance. I don't understand how can i improve this speed and performance. 
Please suggest me best way

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/panels/installation#improving-filament-panel-performance
